README.md

```diff
@@ -112,3 +112,52 @@ You can also set the stream options by using the following environment variables
 - `INTERFACE_NAME`
 - `INTERVAL_BTW_SAMPLES`
 - `SCALE`
+
+## Docker
+
+### Build the Container
+
+First, ensure you have Docker installed on your machine, then:
+1. Clone the repository containing the Stream Rust Test code
+2. Navigate to the root directory of the repository
+3. Run the build script to create the Docker image:
+   ```sh
+   ./scripts/docker/build.sh
+   ```
+
+### Run the Container
+
+To run the container with your configuration file:
+1. Ensure you have defined your astarte configuration file `config.toml`
+2. Run the Docker container, mounting the configuration file:
+   ```sh
+   docker run -v /path/to/your/config.toml:<MOUNT_TO_THIS_PATH> -e ASTARTE_CONFIG_PATH="<MOUNT_TO_THIS_PATH>" stream-rust-test:latest
```

Comment:
Instead of letting the user define both the config path and the mount point it would be easier to provide a default path for the configuration **in** the container.
This way the command would look something like:
```sh
docker run -v /path/to/your/config.toml:/etc/stream-rust.toml stream-rust-test:latest
```

Reply:
The idea here is that the application can be either run using the docker container or with `cargo run`. Therefore, although it makes sense to use the absolute path `/etc/config.toml` when using the container, it gives less flexibility when running the application with cargo. 

Btw, I think it's better to have this conversation in #13

Reply:
Oh sorry, commentend on the wrong pr, i'll move there.